apollo-compiler/src/test/graphql/com/example/merged_include/TestOperation.graphql

```diff
@@ -0,0 +1,8 @@
+query TestQuery {
+    hero {
+        name
+    }
+    hero @include(if: false) {
+        id
```

Comment:
I'm curious what will happen if we add duplicate `name` field? I have assumption that it generator fails in this case

Reply:
Well, for this specific case, it doesn't change anything to the codegen since the `@include(if: false)` prunes any selection in `hero {}`